custom-recipes/timeseries-forecast-1-train-evaluate/recipe.json

```diff
@@ -292,6 +292,87 @@
             "description": "Statistical models",
             "visibilityCondition": "['choose_algorithms', 'customize_algorithms'].includes(model.forecasting_style)"
         },
+        {
+            "name": "season_length_min",
+            "label": "Seasonality",
+            "description": "Length of the training data season to use in statistical models.",
+            "type": "INT",
+            "defaultValue": 60,
```

Comment:
For minute frequency, do we want to say default 60 or default 1 (no seasonality). For sensor data or high-frequency finance data, I assume seasonality has not impact.

Reply:
i'll use 1 as default, I agree about minutes seasonality that don't really make sense

Reply:
b8c967e